modules/beopBidAdapter.js

```diff
@@ -130,6 +131,7 @@ function beOpRequestSlotsMaker(bid) {
     pid: getValue(bid.params, 'accountId'),
     nid: getValue(bid.params, 'networkId'),
     nptnid: getValue(bid.params, 'networkPartnerId'),
+    sgs: getValue(bid.params, 'permutive'),
```

Comment:
permutive adapter sets segments in the ortb2 object, publishers shouldn't need to be setting them as params.